mmcls/apis/train.py

```diff
@@ -222,6 +222,7 @@ def train_model(model,
             **loader_cfg,
             'shuffle': False,  # Not shuffle by default
             'sampler_cfg': None,  # Not use sampler by default
+            'drop_last': False,  # Not drop last by default
```

Comment:
The default behavior of PyTorch should be `drop_last=False`, is this line necessary?